packages/dev/docs/src/client.js

```diff
@@ -182,6 +184,12 @@ ReactDOM.render(<>
   <ThemeSwitcher />
 </>, document.querySelector('.' + docsStyle.pageHeader));
 
+let pathToPage = document.querySelector('[data-github-src]').getAttribute('data-github-src');
+ReactDOM.render(<Button variant="primary" isQuiet elementType="a" href={encodeURI(`https://github.com/adobe/react-spectrum/tree/main/${encodeURI(pathToPage)}`)} target="_blank">
+  <LinkOut />
+  <Text>Edit this page</Text>
+</Button>, document.querySelector('#edit-page'));
```

Comment:
Why does this need to be client rendered? Why not an `a` tag with button css on it?

Reply:
honestly, it's just how old this PR is I think, we've talked a lot about links vs buttons in the last year, I'll go ahead and update
as for "why", i think it makes it a bit more inviting and we've had a number of issues (not huge, but also not zero) where people point out issues in our docs, this just makes it a little easier for people to actually make the change.